src/pyasm/widget/web_wdg.py

```diff
@@ -1156,8 +1154,20 @@ def __init__(my, **kwargs):
     def get_display(my):
         name_label = my.kwargs.get('name_label')
         password_label = my.kwargs.get('password_label')
+        override_background = my.kwargs.get('override_background') == "true"
+        override_logo = my.kwargs.get('override_logo') == "true"
+        override_company_name = my.kwargs.get('override_company_name') == "true"
+        override_textfield = my.kwargs.get('override_textfield') == "true"
+        override_login = my.kwargs.get('override_login') == "true"
+        bottom_help = my.kwargs.get('bottom_help') == "true"
+        bottom_link = my.kwargs.get('bottom_link')
```

Comment:
bottom_help and bottom_link should be part of one kwarg.. Let's just call it bottom_link.. and  it will be a json formatted dictionary:

{"User Documentation": "www.yahoo.com"}

for example  and you load it by
from pyasm.common import jsonloads
if bottom_link:
       bottom_info = jsonloads(bottom_link)